### DISCUSSION TITLE
T-Echo hangs with red light. Seems not to turn on

### DISCUSSION THREAD

**Discussion by Drake008** [meshtastic/firmware#3862] [Troubleshooting]
### Category

Hardware Compatibility

### Hardware

T-Echo

### Firmware Version

2.33

### Description

Hi there,

my T-Echo hangs with just the red charging light. Even clicking the reset button twice to get a Drive in the explorer wont work.
When I hold the reset button there is a purple light with the red light on.

Anyone with the same problem? Is it bricked?

**Comment by thebentern** [comment]
Sounds like a flash / filesystem issue. You need to erase the device and load a fresh firmware.
https://flasher.meshtastic.org/
Select T-Echo and click the Trashcan icon and follow the instructions

**Comment by Drake008** [comment]
Tried it and the device is not shown in the selection list (for entering the DFU Mode). Whenn I click two times the reset button I get a purple light for a fraction of a second and thats it.

So I do not get into dfu mode.

**Comment by JimTheCowboy** [comment]
This seems to be another duplicate of #5839 
Should be mostly solved with Firmware 2.5.20 beta and upwards and requires a NRF factory reset.
https://github.com/meshtastic/firmware/releases/tag/v2.5.20.4c97351

@fifieldt  this seems to be the last one =)
